magento/magento2
# Shubham 18724

Description (*)
There was a issue in display tax percent if select any locale where comma(,) use instead of decimal (.) for an example Germany.

Fixed Issues (if relevant)
#18724 : Display tax percent in current locale number format

Manual testing scenarios (*)
1: login magento admin
2: Set in admin locale Germany or other European country where decimal character is comma
3: In Admin > Stores > Configuration > Sales > Tax > Shopping Cart Display Settings > Display Full Tax Summary = Yes and in the other section "Orders, Invoices, Credit Memos Display Settings"
4: Set a tax percent in admin with decimals like 50.1.
5: Add any product to cart and goto cart page.
6: Check order summary.

Contribution checklist (*)
  Pull request has a meaningful description of its purpose
  All commits are accompanied by meaningful commit messages
  All new or changed code is covered with unit/integration tests (if applicable)
  All automated tests passed successfully (all builds on Travis CI are green)
